internal/integration/testdata/sqlite/cli-migrate-lint-add-notnull.txt

```diff
@@ -0,0 +1,19 @@
+atlas migrate lint --dir file://migrations --dev-url URL --latest=2 > got1.txt
+cmp got1.txt expected1.txt
+
+-- migrations/1.sql --
+CREATE TABLE users (id int);
+
+/* Adding a not-null column without default to a table created in this file should not report. */
+ALTER TABLE users ADD COLUMN c1 int NOT NULL;
+
+-- migrations/2.sql --
+ALTER TABLE users ADD COLUMN c2 int NOT NULL;
+
+ALTER TABLE users ADD COLUMN c3 int NOT NULL DEFAULT 1;
+
+-- expected1.txt --
+Data dependent changes detected in file 2.sql:
+
+	L1: Adding a non-nullable "int" column "c2" will fail in case table "users" is not empty
```

Comment:
Wondering if we should also include the SQL in all reports here: 

```
L1: ALTER TABLE users ADD COLUMN c1 int NOT NULL
    Adding a non-nullable "int" column "c2" will fail in case table "users" is not empty
    
...
```

Reply:
I don't know yet. I plan to improve the printing soon to look more like the `migrate apply` output. I will ask @hilakashai to help with its design.